### DISCUSSION TITLE
Are images expected to be between [0, 1] or [0, 255] in data pipeline

### DISCUSSION THREAD

**Discussion by dienachtderwelt** [open-mmlab/mmediting#1599] [General]
While looking at codes related to Real-ESRGAN, I've noticed that random_degradations switched to do transformations on 0-255 images in dev-1.x branch instead of 0-1 images in the master branch. However in the config file the RescaleToZeroOne is still there in the data pipeline BEFORE all the degradations. Also in dev-1.x branch models are using a new config data_preprocessor.
Is there a change in design that from ver. 1.0 and later data transforms should be performed on 0-255 images? Or it just depends on how the user writes the config file?
I haven't checked all the CONFIGs and TRANSFORMs but the conflict between the two really exists in Real-ESRGAN.

**Comment by dienachtderwelt** [comment]
@Z-Fran Hi a kind invitation here lol. Would you mind take a look at this? Not quite sure if it's a single or a higher level problem. Not very familiar with the whole project architecture.

**Reply by Z-Fran** [reply]
This PR will fix this bug. All images are [0-255]. https://github.com/open-mmlab/mmediting/pull/1583

**Reply by dienachtderwelt** [reply]
> This PR will fix this bug. All images are [0-255]. #1583

Oh sorry didn't see that lol

**Comment by Z-Fran** [comment]
This PR will fix this bug. All images are [0-255]. https://github.com/open-mmlab/mmediting/pull/1583
